src/Components/Components/src/Sections/HeadContent.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Components.Rendering;
+
+namespace Microsoft.AspNetCore.Components.Sections
```

Comment:
I think we should keep `Sections` out of the namespace since that's not a public API concept yet. Ideally I agree it would be in `Microsoft.AspNetCore.Components.Web`, both package and namespace.

Reply:
That sounds good to me. We'll just need to decide if it would be acceptable then to have an IVT from `Microsoft.AspNetCore.Components.Web` to `Microsoft.AspNetCore.Components` so we can attach the `SectionRegistry` to `Dispatcher`. Alternatively, we can use the previous approach with a `ConditionalWeakTable` but performance concerns were expressed with that.

Reply:
From our discussion just now, it sounded conclusive enough that we were all happy to add the IVT.

No pressure to do that refactoring just now though - it could be done after preview 7.